tests/org.eclipse.jface.tests/src/org/eclipse/jface/tests/viewers/TableViewerTest.java

```diff
@@ -191,4 +191,16 @@ public void testRemove() {
 
 	}
 
+	public void testContains() {
```

Comment:
This test fails for a "virtual" table viewer: org.eclipse.jface.tests.viewers.VirtualLazyTableViewerTest.testContains.